tests/cases/standalone/common/select/dummy.result

```diff
@@ -103,3 +103,11 @@ DROP TABLE test_unixtime;
 
 Affected Rows: 1
 
+select Interval '1 year 2 months 3 days 4 hours 5 minutes 6 seconds 100 microseconds';
+
++---------------------------------------------------------+
+| IntervalMonthDayNano("1109194275255040973236744059552") |
++---------------------------------------------------------+
+| 0 years 14 mons 3 days 4 hours 5 mins 6.000100000 secs  |
++---------------------------------------------------------+
+
```

Comment:
There is something confusing. When I start greptimedb with `standalone` mode, and use Mysql client connect it ,it will show `iso_8601` format as I set.https://github.com/GreptimeTeam/greptimedb/blob/354ef9e9fd1cb8c38b6b2b4bba0938d7aea26fd1/src/servers/src/mysql/writer.rs#L185
But sqlness generate diff result, it seems like `postgres verbose` format, I want to know why this is happening, is it because of diff client? Should I unify the format?
```
mysql> select Interval '1 year 2 months 3 days 4 hours 5 minutes 6 seconds 100 microseconds';
+---------------------------------------------------------+
| IntervalMonthDayNano("1109194275255040973236744059552") |
+---------------------------------------------------------+
| P1Y2M3DT4H5M6.0001S                                     |
+---------------------------------------------------------+
1 row in set (0.00 sec)
```